setup.py

```diff
@@ -42,6 +47,8 @@ def main():
                         help="Path to build directory. Default: 'build'")
     parser.add_argument("--install-pcl-deps", action='store_true',
                         help="Install dependencies for PCL extension and exit")
+    parser.add_argument("--install-ros2-deps", action='store_true',
+                        help="Install dependencies for ROS2 extension and exit")
```

Comment:
Please update the documentation [here](https://github.com/RobotecAI/RobotecGPULidar/blob/develop/docs/Ros2Extension.md).